model/src/main/java/org/projectnessie/model/Operation.java

```diff
@@ -48,6 +49,14 @@
   @NotNull
   ContentsKey getKey();
 
+  @Schema(
+      type = SchemaType.OBJECT,
+      title = "Put-'Contents'-operation for a 'ContentsKey'.",
+      description =
+          "Add or replace (put) a 'Contents' object for a 'ContentsKey'. "
+              + "If the actual table type tracks the 'global state' of individual tables (Iceberg "
+              + "as of today), every 'Put' must be accompanied by a 'PutGlobal' to update the global "
+              + "state.")
```

Comment:
If I understand correctly, for an Iceberg `Put` operation `Contents` would be an instance of `IcebergTable`, right? Since, `IcebergTable` already tracks both global and per-branch state, why do we require a separate `PutGlobal` operation? Could we not split the per-branch data and global data on the server side when we're about to store it? :thinking:

Reply:
> hy do we require a separate PutGlobal operation? 

There is no PutGlobal operation (see updated PR description).

> Could we not split the per-branch data and global data on the server side when we're about to store it?

That's what will happen - split on the server side.

Reply:
the doc is out of date

Reply:
argh - fixed